frontend/src/css/modules/main-header.scss

```diff
@@ -39,6 +39,12 @@
             color: $highlight;
             background-color: rgba(0, 0, 0, 0);
         }
+        a.main-header-link {
```

Comment:
@jainarunarunjain you just need to add a media query for this. Try to add with screen size less than 992px.